MobileCenterCrashes/MobileCenterCrashes/Internals/Model/MSAppleErrorLog.m

```diff
@@ -12,6 +12,7 @@
 static NSString *const kMSOsExceptionAddress = @"os_exception_address";
 static NSString *const kMSExceptionType = @"exception_type";
 static NSString *const kMSExceptionReason = @"exception_reason";
+static NSString *const kMSSelectorRegisterValue = @"selector_register_value";
```

Comment:
I think the `selectorRegisterValue` property should be checked in the `isEqual` methods.